app/views/posts/_wc2025.html.erb

```diff
@@ -0,0 +1,71 @@
+<% return if @video_id.blank? %>
```

Comment:
You should do this check one level up, i.e. `render 'wc2025' unless @video_id.blank?`